test/mongocontroller/controller.go

```diff
@@ -80,18 +96,8 @@ func New(p Params) (*Controller, error) {
 	if err != nil {
 		return nil, err
 	}
-	res := client.Database("foo").RunCommand(nil, map[string]int{"buildinfo": 1})
-	if res.Err() != nil {
-		return nil, res.Err()
-	}
-	var doc map[string]interface{}
-	err = res.Decode(&doc)
-	if err != nil {
-		return nil, err
-	}
```

Comment:
The point of this section was not only to get the version but to test the connection

Reply:
I assume I just need to keep L83-L86 then for testing connection.

Reply:
yep. I suppose you could compare the version to what you expect but that doesn't seem super useful

Reply:
I guess I would at least check that the returned structure is as you expect

Reply:
👍

Reply:
:+1: